src/mainwindow.cpp

```diff
@@ -293,6 +325,64 @@ MainWindow::MainWindow(QWidget *parent) :
         ui->resultsTabWidget->setTabToolTip(i, ui->resultsTabWidget->widget(i)->toolTip());
     }
 
+    QSettings settings(QStringLiteral("KDAB"), QStringLiteral("Hotspot"));
```

Comment:
please extract this stuff into a separate function, to make it clear that this code is only about code navigation / settings